routes/applicationRoutes.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable indent */
```

Comment:
Linting rules shouldnt be removed unless there is a really good reason. I dont know where we stand on linting right now, so just send it 😉

Reply:
There hasnt been any instructions on how to setup the formatting/linting correctly so i dont blame anyone for this. As far as i could tell the correct setup is having the eslint extension installed and setting eslint as the formatter. This means other formatters like prettier shouldnt be used as they arent configured for the project and conflict with eslint because of it.